### PR TITLE
Removed deprecated functions calls in GMod code

### PR DESCRIPTION
**Pull request by Kefta**
I didn't touch any of the panel functions -- except SetKeyBoardInputEnabled -- since if they're removed, they won't explicitly error. Panel:Refresh even seems to be manually called in a few places as well. I also did not convert over DListBox, DModelSelect, and DPanelList.

I left spawnmenu.SwitchToolTab untouched since I'm hoping to back through later and fix it.

Add Entity:SetNetworkedBonePosition, Entity:SetNetworkedVarProxy, and Entity:GetNetworkedVarProxy to the deprecated list.

### CODE REVIEW

**Comment by wiox** [comment]
@svdm appeared against removing util.IncludeClientFile in the past.

**Comment by thegrb93** [comment]
Is UTIL_IsUselessModel a remnant from gmod9? Reminds me of it. So long gmod9.

**Comment by Kefta** [comment]
@wiox I removed it since it seems to be an exact copy of IncludeCS. I can leave it if he wishes

**Comment by Bo98** [comment]
There's still merge conflicts here - probably with the halo file.

File: garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua [hunk 1]
```diff
@@ -20,13 +20,15 @@ end
...
+function ENT:SetNWBonePosition( i, Pos, Angle )
 
-	self:SetNetworkedVector( "Vector" .. i, Pos )
-	self:SetNetworkedAngle( "Angle" .. i, Angle )
+	self:SetNWVector( "Vector" .. i, Pos )
+	self:SetNWAngle( "Angle" .. i, Angle )
 
 end
 
+ENT.SetNetworkedBonePosition = meta.SetNWBonePosition
```
Comment (Bo98) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: Meta is nil here.

Comment (Kefta) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: I don't know why I put meta there; will fix that along with the halo conflict (which is not changing for some reason)

Comment (robotboy655) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: We don't need that alias.

**Comment by robotboy655** [comment]
I can't mege this PR, and you should move TTT changes to a separate PR.

File: garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua (hunk 1 above)
Comment (Bo98) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: Will it not break some addons?

Comment (robotboy655) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: We don't need that function change either.

**Comment by Kefta** [comment]
I'll remake the PR and split it

File: garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua (hunk 1 above)
Comment (Kefta) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: Why not? Get/SetNetworkedVarProxy was changed to NW; this is the only "Networked" function left

Comment (robotboy655) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: This fucking is nothing to do with Set/GetNW*

Comment (robotboy655) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: Oh, wait, it only works for gmod_ghost.

Comment (Kefta) [inline garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua]: So don't rename it/don't touch it at all?

**Comment by robotboy655** [comment]
Remake it is.

**Pull request closed without merging**